### DISCUSSION TITLE
user-guide/faqs/update-translators

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#191] [文档评论区]
# user-guide/faqs/update-translators

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/faqs/update-translators

**Comment by northword** [comment]
> 建议取消勾选“自动检查转换器和样式的更新”前面的复选框，这样可以避免非官方维护中文转换器被意外替换为官方的版本。如果遇到了文献抓取失败的问题，可再按照本文档的方法更新。

@jiaojiaodubai 这个会被替换掉吗？它应该会比较 `updateTime` 吧

**Comment by jiaojiaodubai** [comment]
翻到了它的代码实现，对照 `lastUpdated`，设置是否更新（`extract`）：
https://github.com/zotero/zotero/blob/a6076ce76c2401b4313d2905afac74836834681c/chrome/content/zotero/xpcom/schema.js#L1182-L1184
https://github.com/zotero/zotero/blob/a6076ce76c2401b4313d2905afac74836834681c/chrome/content/zotero/xpcom/schema.js#L1202C1-L1205C7
所以启用这个选项是安全的。

**Comment by matatazi** [comment]
在更新【每一个浏览器中的 Zotero Connector】时，发现无论点击几次，每次saved translators数量都明显偏少（75），已经尝试过数次重启zotero和浏览器，都没有用。目前我的zotero只能实现网页抓取功能，而无法抓取知网、wos等网站的论文元数据及PDF。请问有什么方法能解决呢？

**Reply by jiaojiaodubai** [reply]
编辑—设置（首选项）—高级—数据库维护—重置转换器，等10s左右，然后继续”方法1“的步骤2和步骤3。

**Reply by matatazi** [reply]
按照您的方法重复了几遍，依旧没有效果😭我尝试了谷歌、Safari、edge和火狐，无论是中文网站还是英文网站都发生了同样的问题（只能抓取网页），重新下载zotero、茉莉花插件也都没有任何用处。在网页上抓取文献元数据时，zotero可以出现正常的“文本图标”，但是等待一段时间以后仍然只能抓取到网页。另外，昨天在反复尝试您的方法时，zotero弹出过两次和API相关的网页，但是我没有查到相关的信息就都选择了默认，我不清楚这和我目前遇到的问题有没有关系诶

**Reply by jiaojiaodubai** [reply]
Translator 不保证一定能下载附件，请不要把 Connector 作为下载器看待。

**Reply by matatazi** [reply]
问题解决了！谢谢你～

**Reply by r1ng13** [reply]
请问你是怎么解决的？

**Comment by matatazi** [comment]
您好，这是问题相关的截图～

---- Replied Message ----
| From | ***@***.***> |
| Date | 06/10/2024 15:56 |
| To | zotero-chinese/wiki ***@***.***> |
| Cc | matatazi ***@***.***>,
Comment ***@***.***> |
| Subject | Re: [zotero-chinese/wiki] user-guide/faqs/update-translators (Discussion #191) |

编辑—设置（首选项）—高级—数据库维护—重置转换器，等10s左右，然后继续”方法1“的步骤2和步骤3。

**Comment by 2973236988** [comment]
问题：抓取的pdf没有自动命名为论文标题？

**Reply by jiaojiaodubai** [reply]
对于Z6，zotfile之类的插件可能会帮你重命名
对于Z7，软件中显示的是附件的类型（如“全文”、“支撑材料”、“实验数据”、“修正”等），PDF的文件名默认更改

**Reply by northword** [reply]
https://zotero-chinese.com/user-guide/add-attachments#%E9%99%84%E4%BB%B6%E5%90%8D-vs-%E6%96%87%E4%BB%B6%E5%90%8D

**Reply by 2973236988** [reply]
非常感谢

**Comment by yuyuyu0803** [comment]
你好，我的zotero的插件界面里面已经下载茉莉花了，但是没有茉莉花设置的那个选项，也抓取不到知网数据怎么办

**Reply by jiaojiaodubai** [reply]
详见百科全书“常见问题”一章关于插件的说明

**Comment by handong66** [comment]
您好！搜索“更新文献信息”进入此页面，想请问Zotero 7现在有没有像6一样可以更新文献信息的功能。有时添加的条目信息不全，手动添加doi后希望重新抓取信息。谢谢。

**Comment by createrzhu** [comment]
问题：win 11，zotero 7.0 7.0.4-beta.3+5bd5bfa34 (64-bit)中PDF无法添加annotation，下划线和填充等annotation无显示，左侧状态栏也无显示，反而在chartero中滚动条可以显示annotation大致范围。重装zotero7、禁用插件也无法解决（尤其是style）。未查到相关解决办法。请问如何解决？我两台电脑一台是win10 zotero6一台是win11zotero7，win10可以正常使用，二者用zotero+infinicloud进行连接，请问应该如何排查问题？

**Reply by createrzhu** [reply]
问题已经解决，还是插件问题pdffigure插件和annotation冲突所致。打扰了

**Comment by ANNY0730** [comment]
想问一下，我现在处于抓取了文献条目也抓取了网页，建了两个条目，可以怎样合并或者集成么

**Reply by jiaojiaodubai** [reply]
删掉网页，无需合并

**Comment by Whr99088** [comment]
严格按照本网页方法尝试多次，知网界面抓取时full pdf为灰色，仅能抓取一个网页

**Comment by Whr99088** [comment]
zotero 7.0.11，谷歌浏览器，插件也为最新版，茉莉花1.0.3

**Reply by Whr99088** [reply]
2.18已恢复

**Comment by Simoody** [comment]
你好，我的Zotero在抓取条目时会弹出爱斯维尔的机器人验证，但是无法验证成功导致抓取不了pdf，只有条目和网页快照，请问这个要怎么解决？

**Reply by Simoody** [reply]
点击那个机器人验证一直转圈圈，然后又会重新让验证，然后重复一直这样最后它自动关闭这个界面

**Reply by hihule** [reply]
我也是

**Reply by wakewon** [reply]
ScienceDirect 的 PDF 抓取问题目前无解。参见：[Zotero Forum: Zotero is not saving PDFs from ScienceDirect](https://forums.zotero.org/discussion/109940/zotero-is-not-saving-pdfs-from-sciencedirect#latest)

**Comment by createrzhu** [comment]
zoplicate插件，可以解决


ANNY0730 ***@***.***> 于2025年1月7日周二 05:00写道：

**Comment by sylvia-303** [comment]
点了zotero connector，但是没有save to zotero选项，这是为什么呀

**Reply by wakewon** [reply]
有什么？

**Comment by Myqweq** [comment]
我真的很难过，非常需要救命，我的connector不管哪个浏览器，都不行，在pubmed上都只能保存网页，明明两天前都还是好的，可以有个黄色的小文件夹显示，可以有弹窗保存；这两天怎么都不行了，换了各种办法，中文社区我看到的所有办法也都试过了，就是不行；然后知乎网页浏览就可以有小黄色文件夹保存知乎文章，真的想不明白为什么pubmed不行，因为既然知乎网页浏览出现了，某种程度上说明浏览器没问题，插件没问题，所以是pubmed下了connector的禁令吗？？  求救！！！！

**Reply by Beyonder0721** [reply]
可以清除浏览器pubmed的cookies和浏览记录试一下

**Comment by Myqweq** [comment]
真的可以了！！！  谢谢大神！！！！！！

**Comment by hihule** [comment]
direct to science又用不了了，验证没用，要一直验证

**Reply by Simoody** [reply]
我也是，一直验证，最后它自己关掉了，抓不到pdf

**Reply by jiaojiaodubai** [reply]
是网站有意要防，这边是没法解决的

**Reply by wakewon** [reply]
ScienceDirect 的 PDF 抓取问题目前无解。参见：[Zotero Forum: Zotero is not saving PDFs from ScienceDirect](https://forums.zotero.org/discussion/109940/zotero-is-not-saving-pdfs-from-sciencedirect#latest)

（不是 direct to science 哈）

**Comment by Whr99088** [comment]
最近打开知网超级卡，有的时候干脆报错。safari浏览器就很流畅访问知网😢

**Reply by wakewon** [reply]
禁用所有浏览器插件后再尝试，特别是一些脚本。

**Comment by siu-123** [comment]
想问下为什么现在抓取不了pdf？像在IEEE上之前是可以的，现在不行了

**Reply by jiaojiaodubai** [reply]
Connector 抓取时，进度窗口显示红叉吗？这不是中文转换器负责的网站，建议先检查账户权限，然后尝试手动下载更新浏览器token，还不行再到[官方仓库](https://github.com/zotero/translators)反馈。

**Reply by siu-123** [reply]
不是红叉，而是translator错误之类的

**Comment by DWGqaz123** [comment]
地点识别不出来怎么办

**Reply by jiaojiaodubai** [reply]
请提供测试链接。

**Comment by Vanellope0910** [comment]
您好！我的Zotero在尝试了所有的办法以后，依旧读取不到文献信息，显示：
Linter for Zotero
× 监测到您导入了一个 WebPage 条目，其 URL 中包含了主要学术出版商的域名，\n导入可能存在异常，请确认!
请问有什么办法解决吗？

**Reply by wakewon** [reply]
当前这篇文档就是解决这一问题的，请务必仔细阅读，从头到尾认真排查。

**Comment by Telperiwi** [comment]
你好，以往使用插件在【知网】批量抓取文献时可以有个黄色的小文件夹，但是最近突然不行了，只能抓取到网页快照。使用的是Zotero7和最新的茉莉花插件。
按方法更新了很多次转换器，插件也卸了又装，就是在知网上不行。知网的单个文献页面可以正常抓取，wos的外网文献可以用黄色文件夹图标批量抓取。只有知网的批量抓取不行，没有文件夹图标，只有网页。按下面的建议清除了四周内的cookies和浏览记录也没用。

**Reply by wakewon** [reply]
这是知网的变更导致的，目前无解。不要在检索页进行抓取，无论是批量还是单个都不行。

此外，任何时候都不建议批量抓取，容易被风控。

**Reply by Telperiwi** [reply]
非常感谢！我询问周围的人发现他们也是这样的情况，我还以为我的插件坏了。

**Comment by dowella** [comment]
你好，上述更新完之后，知网文献日期仍然只能显示2025，不能显示日月，请问要怎么做

**Reply by wakewon** [reply]
联系知网，让他们在方便抓取的地方提供足够充分的信息。

**Comment by l-lys-511** [comment]
为什么我的中文转换器更新显示：成功0，失败0，跳过93？这是更新了还是没更新？
还有就是更新浏览器里的zotero connecter translator时，右键只能保存网页，打不开“选项”是为什么？

**Reply by wakewon** [reply]
1. 跳过=无需更新。
2. 重装 connecter，以及不同浏览器操作不同，详见前面文档。

**Comment by TianzheNyx** [comment]
您好，上面的全部进行了一遍包括手动替换js文件但是抓中文文献的时候Full text pdf一直是灰色的，没有红叉，请问下我该怎么解决

**Reply by wakewon** [reply]
开头基础知识先看下

**Comment by Xilinjin** [comment]
为什么我出现这样：使用 Wiley Online Library 保存时发生错误。改为尝试用 Atypon Journals 保存。
使用 Atypon Journals 保存时发生错误。改为尝试用 Embedded Metadata 保存。 外文几乎全都是抓取失败。

**Reply by wakewon** [reply]
按照前面文档说的挨个处理，仍然不行的话去Zotero官方转换器仓库提issue。中文社区没有海外文献抓取转换器的维护者。

**Comment by plain-stu** [comment]
2025年10月31日发现很多文献抓取不到具体日期，仅能抓取到所在年份。回看以前抓取的文献，部分期刊的月日信息似乎也都丢失。问题发生在IEEE Transactions on Ultrasonics, Ferroelectrics, and Frequency Control、IEEE Transactions on Instrumentation and Measurement、Journal of Lightwave Technology（同为IEEE的）、一些中文期刊，但并不是该期刊的所有文章都出现了此问题。

问题文章doi：
10.1109/JLT.2022.3182627
10.1109/TIM.2025.3626107
10.1109/TIM.2025.3612558
10.1109/JSEN.2025.3615130

**Reply by wakewon** [reply]
海外文献的抓取是官方维护的，中文社区没有相关的维护者。可以去官方转换器仓库提issue反馈。

**Comment by aichiija** [comment]
使用 CNKI 保存时发生错误。改为尝试用 DOI 保存。
使用 DOI 保存时发生错误。改为尝试用 Save as Webpage 保存。
这个怎么办啊  而且未在该目录下发现pdftk程序

**Reply by wakewon** [reply]
你的茉莉花非常古早了，仔细阅读前面文档的各个说明。

**Comment by jiaojiaodubai** [comment]
试火狐



---Original---
From: ***@***.***&gt;
Date: Thu, Dec 25, 2025 18:50 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zotero-chinese/wiki] user-guide/faqs/update-translators(Discussion #191)




 
使用 CNKI 保存时发生错误。改为尝试用 DOI 保存。
 使用 DOI 保存时发生错误。改为尝试用 Save as Webpage 保存。
 这个怎么办啊  而且未在该目录下发现pdftk程序

**Comment by ChenSiming0428** [comment]
在对文章进行修改时，文末引用的文献突然消失了，只剩一篇，该怎么找回来？

**Comment by 1428329975** [comment]
想问一下抓取的文献题目中有上下角标，在zotero中角标变成题目格式，有的题目还会出现#这种的代码，请问应该怎么设置

**Reply by jiaojiaodubai** [reply]
给截图和链接，否则无法作答

**Reply by 1428329975** [reply]
![Snipaste_2026-04-10_10-14-31](https://github.com/user-attachments/assets/6f4fdef8-7dbb-42f1-82fe-3778ed2bfb67)
![Snipaste_2026-04-10_10-14-53](https://github.com/user-attachments/assets/1c49175b-17bf-4b39-98cf-d48c6bed682f)

**Comment by CNZhou-03** [comment]
您好，更新完zotero9之后，知网抓取不了pdf显示full text pdf一个红叉，但是可以正常显示条目信息这是什么原因呢

**Comment by yunfeibai309-code** [comment]
更新最新版本后，茉莉花插件没有办法抓取知网的条目信息（连文献名称都不会有，也没有snapshot，万方数据库可以抓取），在通过pdf导入条目的时候也会跳出来知网的验证界面，怎么办呢？有尝试上面的步骤

**Reply by wakewon** [reply]
从网页抓取与茉莉花无关，只与转换器有关。我刚刚测试了一下，在详情页里，无论是条目还是PDF抓取都是正常的。文档里的方法已足够，确保完全按照文档说明做完了每一个方案。

通过PDF导入跳人机验证的话，你需要在弹出的窗口做人机验证。知网有各种各样反爬防护，插件会尽力适配，但如果遇到这种人机验证必须得你人工来完成，不可自动跳过。如果关了的话大概率会检索失败。

**Comment by rjn5979g6h-maker** [comment]
请问Zotero 9 版本中，茉莉花无法抓取知网文件，浏览器的插件已更新，茉莉花插件中转换器列表是空的，抓去源当中的选择数据源中国知网也显示是灰色的，这该如何处理呢？
